component-config/secure-api-gateway-ob-uk-common.env

```diff
@@ -22,6 +22,7 @@ STEPS_PR_BUILD_MAVEN_COMMANDS=mvn -B clean package --file pom.xml
 STEPS_PR_BUILD_MAVEN_PROJECT=true
 STEPS_PR_DEPLOY_CHANGES=false
 STEPS_PR_GET_BRANCH_NAME=false
+STEPS_PR_RUN_COPYRIGHT_TESTS=false
```

Comment:
can we enable for this one as well please?

Reply:
Done